addons/project-folders/userscript.js

```diff
@@ -0,0 +1,190 @@
+export default async function ({ addon, global, console, msg }) {
+  let projectColumns;
+  projectColumns = await addon.tab.waitForElement("ul.media-list", {
+    markAsSeen: true,
+  });
+
+  let foldersArray;
+  const projectTab = document.querySelectorAll("li.first")[0];
+  projectTab.childNodes[0].onclick = async () => {
+    projectColumns = await addon.tab.waitForElement("ul.media-list", {
+      markAsSeen: true,
+    });
+    createFolderAreaAndButton();
+    foldersArray = await load();
+  };
+
+  if (document.querySelectorAll("div.folders-container")[0] === undefined) {
```

Comment:
```suggestion
  if (!document.querySelectorAll("div.folders-container")[0]) {
```
The same can be applied elsewhere.